src/routes/paths.ts

```diff
@@ -21,6 +21,7 @@ export const getMapPath = () => '/map';
 export const getVouchingPath = () => '/vouching';
 export const getHistoryPath = () => '/history';
 export const getAdminPath = () => '/admin';
+export const getCreateCirclePath = () => '/createCircle';
```

Comment:
why not just have these be consts when they're not parameterized?

Reply:
case-sensitive paths are a bit less conventional in URLs -- maybe `create-circle` or `new-circle` is better

Reply:
I got that pattern from a library I was using in another project, some paths are parameterized, it could all be constants

Reply:
I changed it to use kabab case, I'd want to look up that other lib before changing to all constants. I think the point is that you can use them in the same way if they are the same type